draft-opik-quic-qmux.md

```diff
@@ -396,17 +396,45 @@ if the transport is blocked by flow or congestion control.
 
 # Version Agility
 
-Unlike QUIC, QMux does not define a mechanism for version negotiation.
+Unlike QUIC version 1, QMux does not include a built-in mechanism for version
+negotiation. However, as new versions of QUIC are specified, there may be a
+desire to define their reliable-byte-stream counterparts in a way that allows
+them to coexist with QMux.
```

Comment:
I don't think "coexist" is the right word here. In the abstract you use "polyfill". I don't know what that word means but the tools in this section enable the same relationship of QMux to other versions as it specifies for QMux to v1. From context, I imagine that's "polyfill".